### PR TITLE
add sprockets recipe

### PR DESCRIPTION
**Pull request by aneta-bielska**

### CODE REVIEW

**Comment by aneta-bielska** [comment]
@nzwsch What do you mean by 'classic style' ? I'm not sure what you are referring to.

**Comment by nzwsch** [comment]
@kgrz Should I merge this?

File: asset_management/sprockets.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+//= require_tree .
+```
+
+It will require all files alphabetically.
+But if you would like to add files individually remove `require_tree .`
+and add each file:
+
+```js
+// app/assets/app.js
```
Comment (kgrz) [inline asset_management/sprockets.md]: Should this be `app/assets/javascripts/app.js`?

Comment (kgrz) [inline asset_management/sprockets.md]: Or  may be the lines below should be pointing to the `javascripts/` folder?

Comment (kgrz) [inline asset_management/sprockets.md]: I tried following the recipe, but I'm getting a 404 for the Javascript file. I'm not sure what I'm doing wrong here. Uploaded the code to: https://github.com/kgrz/sinatra_recipes_sprockets

Comment (nzwsch) [inline asset_management/sprockets.md]: The reason why you got 404 is app.js isn't located in *assets/javascripts/* path or there isn't `environment.append_path "assets"`.
Check out kgrz/sinatra_recipes_sprockets#1.

Comment (kgrz) [inline asset_management/sprockets.md]: Yay! worked. Thanks for pointing that out.

**Comment by nzwsch** [comment]
@aneta-bielska Thank you.

**Pull request merged**